orchestrator/api/tools.py

```
@@ -16,10 +16,10 @@
 import logging
 from pydantic import BaseModel, Field
 
-from database.database import get_db
-from models import Tool, ToolCredentials, ToolConfiguration, ToolUsageLog
-from models import Tool, ToolCredentials, ToolConfiguration, ToolUsageLog
-from utils.logging_adapter import set_request_id
+from core.database.database import get_db
+from core.models import Tool, ToolCredentials, ToolConfiguration, ToolUsageLog
+from core.models import Tool, ToolCredentials, ToolConfiguration, ToolUsageLog
```

Ruff (0.14.7)

Redefinition of unused `Tool` from line 20

Remove definition: `Tool`

(F811)